e2e/app/container.ts

```diff
@@ -63,7 +63,7 @@ export default class Container {
       fp.flow(
         fp.filter<{ shouldWait: boolean; waitFn: () => Promise<void> }>('shouldWait'),
         fp.map((item) => item.waitFn()),
-        fp.concat([button.click({ delay: 10 })])
+        fp.concat([button.click({ delay: 10 }), waitWhileLoading(this.page)])
```

Comment:
Fix a flaky test failure that happened during PR testing. The spinner was not waited for until gone.